engineapi/instance_manager.go

```diff
@@ -211,6 +215,9 @@ func (c *InstanceManagerClient) ReplicaProcessCreate(replica *longhorn.Replica,
 	if replica.Spec.RevisionCounterDisabled {
 		args = append(args, "--disableRevCounter")
 	}
+	if replica.Spec.UnmapMarkDiskChainRemovedEnabled {
```

Comment:
This part needs to rebase to do backward compatibility support

Reply:
It seems that we need to check the backward compatibility for the below option `dataLocality == longhorn.DataLocalityStrictLocal` as well. I will directly include it in this commit.